src/index.html

```diff
@@ -5,8 +5,362 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header header--container">
+
+      <a href="#" >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+          class="header__logo"
+      >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
```

Comment:
don't add names for classes by used tags
```suggestion
          <li class="nav__item">
```